### PR TITLE
Achieving Optimal Accuracy with Multiple Machine Learning Models and Visualization #7

### PR DESCRIPTION
**Pull request by Dishant-garg**
Solved issue #7 
used various techniques like simple imputer strategy mean and strategy most frequent to handle null values 
use label encoder to handle the categorical data
use correlation matrix to visualize the data
cof
after that split data into test set training set 
after that implement various classification techniques and see the accuracy score which one has highest
Screenshot (441)
@Samcoding5854 @darshbaxi pls review the pr

### CODE REVIEW

**Comment by Samcoding5854** [comment]
Good work in solving the issue
The accuracy achieved by your model is one of the highest as compared with all others! Congrats!
I hope the model is not overfitted :)
